### ISSUE TITLE
Mixture of languages in screen-reader section

### ISSUE THREAD

**Issue by jacob1474** [highcharts/highcharts#23744]
#### Expected behaviour
When using the nb-NO.js internationalization file, all chart elements, including accessibility strings in the screen-reader-section, should be fully translated into Norwegian.

#### Actual behaviour
Most of the chart is translated to Norwegian, but some strings within the accessibility screen-reader-section remain in English.

This issue was specifically observed when the xAxis is set to type: 'category'. The terms "categories" and "values" are not translated. However, this problem does not occur with other xAxis types. For example, when the xAxis is set to type: 'datetime', the accessibility section is fully translated into Norwegian as expected.

This results in an inconsistent mixture of English and Norwegian text in the accessibility output, depending on the chart configuration.

#### Live demo with steps to reproduce
Live demo: [JsFiddle](https://jsfiddle.net/7rmos8kb/16/)
##### Steps:
1. Open the JSFiddle link (which uses xAxis: { categories: [...] }).
2. Inspect the chart's HTML (right-click -> Inspect).
3. Search for the div with the class highcharts-screen-reader-region-before-0.
4. Navigate to the child divs and observe the mix of languages (e.g., Norwegian text mixed with the English words "categories" and "values").

#### Screenshot example
This is present in all languages, e.g. Chinese:

<img width="345" height="199" alt="Image" src="https://github.com/user-attachments/assets/b2324457-50e1-451e-b055-14e36bb9fa7e" />

#### Product version
Highcharts Core 12.4.0

#### Affected browser(s)
Not browser dependent

**Comment by magdalena-gut** [comment]
Hi @jacob1474, thank you for reporting this!
I couldn’t find the `screen-reader-section` class in the provided demo on my end. In the `screen-reader-region-` classes, everything seems to be translated correctly.Could you please double-check the mentioned class? Also, any additional details or screenshots would be very helpful in identifying the source of the issue. Thanks!

**Comment by jacob1474** [comment]
Hi @magdalena-gut, sorry for the mix-up. I have updated the original issue with the correct class. Also I added a screenshot from HTML code.

**Comment by magdalena-gut** [comment]
@jacob1474 , thanks for clarifying! I can confirm the issue on my end as well. Since the source of the problem lies within a private function, I can’t provide a quick fix for this, however, it will be added to our backlog.

Internal note:
The getAxisDescription() returns fixed words ('categories', 'time, 'values') for the axes.

**Comment by oysteinmoseng** [comment]
Internal note: Let's just take these words from the lang options instead of using fixed words.
